docs/source/events.rst

```diff
@@ -422,30 +529,809 @@ bar icon.
+they are clicking on things that we don't expect to be clicked. The
+intention is to allow the operator to determine what content should be made
+clickable and what kinds of content are being opened.
+
+The ``content_type`` field that can be included in the payload is currently
+one of:
+
+- ``knowledge_content``: text-based article in a knowledge-app
+- ``knowledge_video``: video based article in a knowledge-app
+- ``knowledge_artwork``: "artwork" card (larger images in a "Gallery" style format)
+- ``undefined``: user clicked on a non-clickable area
+
+:UUID name: ``EVENT_DISCOVERY_FEED_CLICK`` in eos-discovery-feed
+
+.. versionadded:: 3.2.0
+
+Discovery Feed Closed
+~~~~~~~~~~~~~~~~~~~~~
+
+Removed in 4.0.0.
+
+Something is clicked on the Discovery Feed, including content which is not "clickable".
+
+The payload tells us about what users are clicking on generally and whether
+they are clicking on things that we don't expect to be clicked. The
+intention is to allow the operator to determine what content should be made
+clickable and what kinds of content are being opened.
+
+The ``content_type`` field that can be included in the payload is currently
+one of:
+
+- ``knowledge_content``: text-based article in a knowledge-app
+- ``knowledge_video``: video based article in a knowledge-app
+- ``knowledge_artwork``: "artwork" card (larger images in a "Gallery" style format)
+- ``undefined``: user clicked on a non-clickable area
+
+:UUID name: ``EVENT_DISCOVERY_FEED_CLICK`` in eos-discovery-feed
+
+.. versionadded:: 3.2.0
+
+Discovery Feed Opened
+~~~~~~~~~~~~~~~~~~~~~
+
+Removed in 4.0.0.
+
+The Discovery Feed is open.
+
+The payload tells us about the language the user is using when interacting
+with the discovery feed and how they opened it. The intention is to allow
+the operator to determine which languages the discovery feed is most
+popular in and how users generally open the feed.
+
+:UUID name: ``EVENT_DISCOVERY_FEED_OPEN`` in eos-discovery-feed
+
+.. versionadded:: 3.2.0
+
+Disk Space Extra
+~~~~~~~~~~~~~~~~
+
+Removed in 4.0.0.
+
+Total, used and free disk space for ``/var/endless-extra``.
+
+Sent on startup, and every 24 hours.
+
+See `T18445 <https://phabricator.endlessm.com/T18445>`_.
+
+:UUID name: ``EXTRA_DISK_SPACE_EVENT`` in eos-metrics-instrumentation
+
+.. versionadded:: 3.4.3
+
+Entered Demo Mode
+~~~~~~~~~~~~~~~~~
+
+Removed in 4.0.0.
+
+The systems enters demo mode.
+
+Note that the machine ID will be reset just before the system enters demo
+mode, so any metrics collected for this machine ID after this event has
+been fired are metrics for a system that is in demo mode.
+
+Demo mode was removed in EOS 3.9, so this metric will not be seen for
+systems on 3.9.0 or later.
+
+See `T18983 <https://phabricator.endlessm.com/T18983>`_.
+
+:UUID name: ``DEMO_MODE_ENTERED_METRIC`` in gnome-initial-setup
+
+.. versionadded:: 3.2.5
+
+Endless Application Unmaximized
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Removed in 4.0.0.
```

Comment:
This is another "never going to update the SDK to stop sending this" case – but we can stop storing it.